src/app/component/Slider/Slider.component.js

```diff
@@ -277,6 +327,8 @@ export default class Slider extends PureComponent {
             showCrumbs, mix, activeImage, children
         } = this.props;
 
+        this.slidesQty = this.props.children.length;
```

Comment:
There is really no need to do this on each render, put it under `componentDidUpdate`.
Also - `this.slidesQty = children.length`?